package/qemu/qemu.init

```diff
@@ -12,13 +12,31 @@ QEMU_NET_BRIDGE=br0
 QEMU_BLOCK_DEVICE=virtio
 QEMU_VNC_DISPLAY=1
 # QEMU_VNC_TLS="yes" -- Optional, if not set then VNC sessions will be unencrypted (default)
-QEMU_VNC_TLS_DIR=/mnt/kd/ssl/qemu
+QEMU_VNC_TLS_DIR=/mnt/kd/qemu
 
 . /etc/rc.conf
 
+init () {
+
+  if [ "$QEMU_ENABLE" = "yes" ]; then
+    if [ ! -d /mnt/kd/qemu ]; then
+      mkdir /mnt/kd/qemu
+      if [ -d /stat/etc/qemu ]; then
+        cp -a /stat/etc/qemu/* /mnt/kd/qemu/
+      fi
+    fi
+
+    if [ -d /tmp/etc/qemu ]; then
+      rm -rf /tmp/etc/qemu
+    fi
```

Comment:
Since you don't `mkdir /tmp/etc/qemu` anywhere this `-d` test is not needed, but does no harm. Avahi does create a similar directory to handle a special case.